api/src/main/java/edu/cornell/mannlib/vitro/webapp/email/FreemarkerEmailFactory.java

```diff
@@ -150,7 +188,28 @@ private InternetAddress getReplyToAddressFromConfig(ServletContext ctx) {
 	private Session createEmailSession(String hostName) {
 		Properties props = new Properties(System.getProperties());
 		props.put("mail.smtp.host", hostName);
-		return Session.getDefaultInstance(props, null);
+		props.put("mail.smtp.port", emailPort);
+		if (emailPort == TLS_PORT) {
+			props.put("mail.smtp.starttls.enable", "true");
+		}
+		if (emailPort == SSL_PORT) {
+			props.put("mail.smtp.socketFactory.port", emailPort);
+			props.put("mail.smtp.socketFactory.class", "javax.net.ssl.SSLSocketFactory");
+		}
+		Authenticator auth = null;
+		if (!password.isEmpty() && !userName.isEmpty()) {
+			props.put("mail.smtp.auth", "true");
```

Comment:
Do we need "else" part setting mail.smtp.auth to false? Or it is the default value for this property of SMTP protocol?

Reply:
I don't think this is necessary as false is the default value.